src/plinder/core/system/system.py

```diff
@@ -81,31 +103,79 @@ def archive(self) -> Path | None:
 
     @property
     def system_cif(self) -> str:
+        """
+        Path to the system.cif file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "system.cif").as_posix()
 
     @property
     def receptor_cif(self) -> str:
+        """
+        Path to the receptor.cif file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "receptor.cif").as_posix()
 
     @property
     def receptor_pdb(self) -> str:
+        """
+        Path to the receptor.pdb file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "receptor.pdb").as_posix()
 
     @property
     def system_pdb(self) -> str:
```

Comment:
We no longer save system.pdb, only receptor.pdb. IIRC we also wanted to not save system.cif but this is not in yet